### ISSUE TITLE
[질문]LocationBSTNode와 PatientBST사이 관계 질문드립니다.

### ISSUE THREAD

**Issue by BaekSeungGeon** [DSLDataStorage/DS_Project_1_2020_2#2]
고생이 많으십니다. 프로젝트 문서와 스켈레톤코드 사이에 약간 이해가 되지 않는 부분이 있어서 질문드립니다.

프로젝트 문서에는 LocationBST에는 PatientBST의 root노드의 주소값만 저장한다고 되어있는데 스켈레톤코드에는 PatientBST의 주소값이 저장되어 있습니다. 
![image](https://user-images.githubusercontent.com/30014736/93665818-87f10b80-fab4-11ea-8147-34b3d85aa254.png)

![image](https://user-images.githubusercontent.com/30014736/93665815-81629400-fab4-11ea-8538-9d698b9b8f93.png)

LocationBST의 각 노드에는 PatientBST의 주소가 저장되고, PatientBST의 root노드의 초기값은 null이다. 라고 이해하면 될까요?

**Comment by DSLDataStorage** [comment]
네, patient_bst에 속한 데이터가 없는 초기의 경우, patient_bst의 root 노드 초기값은  null 이라고 이해하시면 됩니다.
